.travis.yml

```diff
@@ -22,11 +27,13 @@ addons:
       - g++-4.8
 
 before_install:
-    - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
+  - openssl aes-256-cbc -K $encrypted_e09404c5d59a_key -iv $encrypted_e09404c5d59a_iv -in '.travis/github_deploy_key.enc' -out github_deploy_key -d
```

Comment:
line comments would be really helpful here to give future devs an indication of how to work with this